cpp/src/community/nvgraph_gdf.cu

```diff
@@ -290,11 +285,10 @@ void extract_subgraph_vertex_nvgraph(Graph* gdf_G,
                                               gdf_column* vertices,
                                               Graph* result) {
 
-  CUGRAPH_EXPECTS(gdf_G != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(gdf_G->adjList != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(vertices != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(vertices->data != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(!vertices->valid, "Column must be valid");
+  CHECK_GRAPH(gdf_G);
+  CUGRAPH_EXPECTS(vertices != nullptr, "Invalid API parameter: vertices is NULL");
+  CUGRAPH_EXPECTS(vertices->data != nullptr, "Invalid API parameter: vertice data is NULL");
+  CUGRAPH_EXPECTS(!vertices->valid, "vertices must be valid");
```

Comment:
I think error message here is misleading. This says vertices column should be non-nullable (so, valid should be nullptr).

Reply:
this si another area that will be fixed when gdf_column is removed